src/main/java/org/elasticsearch/action/admin/cluster/repositories/verify/TransportVerifyRepositoryAction.java

```diff
@@ -69,7 +69,7 @@ protected VerifyRepositoryResponse newResponse() {
 
     @Override
     protected ClusterBlockException checkBlock(VerifyRepositoryRequest request, ClusterState state) {
-        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA, "");
+        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA_WRITE, "");
```

Comment:
I don't know, I believe METADATA_READ is probably enough but can't really judge, let's hear from @imotov ? ;)